bindings/java/build.gradle

```diff
@@ -7,7 +7,7 @@ plugins {
 
 archivesBaseName = 'whispercpp'
 group = 'io.github.ggerganov'
-version = '1.4.0'
+version = '1.4.1'

```

Comment:
Why do we need to update the binding version number?

Reply:
I chaged that project io.github.ggerganov whispercpp